.github/workflows/release.yml

```diff
@@ -251,6 +261,14 @@ jobs:
       - name: Build CLI
         run: |
           nix build -L .#cli-release
+      - name: Update versions.json in main
+        uses: ./.github/actions/update_versions_json
+        with:
+          version: "${{ needs.process-inputs.outputs.MAJOR_MINOR_PATCH }}"
+      - name: Commit new versions.json
+        run: |
+          git add versions.json
+          git commit -m "release: update versions.json"
```

Comment:
I think we should use an artifact for the updated versions.json to get this into the other job.